util/fipstools/test_fips.c

```diff
@@ -140,8 +149,128 @@ int main(int argc, char **argv) {
   printf("  got ");
   hexdump(output, out_len);
 
+  EVP_AEAD_CTX_zero(&aead_ctx);
   EVP_AEAD_CTX_cleanup(&aead_ctx);
 
+  /* AES-CCM */
+  OPENSSL_memset(nonce, 0, sizeof(nonce));
```

Comment:
Probably should be consistent in the usage of either using `memset` or `OPENSSL_memset`.

Reply:
Will do, I'll switch `memset` to `OPENSSL_memset` since I believe that's what's being used already elsewhere in the file.